ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -234,7 +234,7 @@ private String location(ImportTableDesc tblDesc, Database parentDb)
 
     LOG.debug("adding dependent CopyWork/AddPart/MoveWork for table "
             + table.getCompleteName() + " with source location: "
-            + dataPath.toString() + " and target location " + tmpPath.toString());
+            + dataPath.toString() + " and target location " + tgtPath.toString());
```

Comment:
For ACID table we always have move task which copies from staging dir which doesn't exist.

Reply:
if src and dest is same move is a no op